### PR TITLE
Redis AI module

### PR DESCRIPTION
**Pull request by danitseitlin**

### CODE REVIEW

File: modules/redis-ai.ts [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+     * @param key 
+     * @param type 
+     * @param data 
+     * @param shape 
+     */
+    async tensorset(key: string, type: TensorType, data?: string[] | Buffer[], shape?: string): Promise<void> {
+        try {
+            let args: (string | Buffer)[] = [key, type];
+            if(shape !== undefined)
+                args = args.concat(['shape', shape])
```
Comment (filipecosta90) [inline modules/redis-ai.ts:26]: <shape> is not a keyword. 
Example command:
```
redis> AI.TENSORSET mytensor FLOAT 2 2 VALUES 1 2 3 4
OK
```
Please check https://github.com/RedisAI/redisai-js/blob/master/src/tensor.ts#L51

Comment (danitseitlin) [reply modules/redis-ai.ts]: @filipecosta90 Thanks, I'll check it out

Comment (danitseitlin) [reply modules/redis-ai.ts]: @filipecosta90 Do you have any example for BLOB data maybe? can't' seem to get it.

Comment (filipecosta90) [reply modules/redis-ai.ts]: yep, here it is:
https://github.com/RedisAI/redisai-js/blob/master/tests/test_client.ts#L153

**Pull request merged**